packages/wagtail/modules/components/Blocks/BannerBlock/index.tsx

```diff
@@ -0,0 +1,37 @@
+'client side'
```

Comment:
is this directive being used?

Reply:
A lot actually. All blocks have to be client-side because I send a list of blocks inside an object to the renderer block. This happens on the server side to prerender the blocks and enhance SEO. The problem is that I can't send a list of server-side components via props. It triggers an error. The components in this list must be client-side components. They are prerendered on the server side anyway (guaranteeing SEO), but they are considered client-side only, which avoids the error.